project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -93,103 +93,109 @@
     </section>
   <?php endif; ?>
 
-  <!--  TOP MENU -->
```

Comment:
@nirgn975 
Please give some help over here of how to implements a _hook_suggestion_
thanks.